sdk/communication/communication-call-automation/src/models/models.ts

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  CommunicationIdentifier,
+  CommunicationUserIdentifier,
+  PhoneNumberIdentifier,
+} from "@azure/communication-common";
+import {
+  CallConnectionStateModel,
+  KnownRecordingChannelType,
+  KnownRecordingContentType,
+  KnownRecordingFormatType,
+  KnownRecordingStorageType,
+} from "../generated/src";
+
+export {
+  CallRejectReason,
+  KnownCallRejectReason,
+  MediaStreamingConfiguration,
+  MediaStreamingTransportType,
+  KnownMediaStreamingTransportType,
+  MediaStreamingContentType,
+  KnownMediaStreamingContentType,
+  MediaStreamingAudioChannelType,
+  KnownMediaStreamingAudioChannelType,
+  CallConnectionStateModel,
+} from "../generated/src/models/index";
+
+/** Properties of a call connection */
+export interface CallConnectionProperties {
+  /** The call connection id. */
+  callConnectionId?: string;
+  /** The server call id. */
+  serverCallId?: string;
+  /**
+   * The source caller Id, a phone number, that's shown to the PSTN participant being invited.
+   * Required only when calling a PSTN callee.
+   */
+  sourceCallerIdNumber?: PhoneNumberIdentifier;
+  /** Display name of the call if dialing out to a pstn number. */
+  sourceDisplayName?: string;
+  /** Source identity. */
+  sourceIdentity?: CommunicationIdentifier;
+  /** The targets of the call. */
+  targets?: CommunicationIdentifier[];
+  /** The state of the call connection. */
+  callConnectionState?: CallConnectionStateModel;
+  /** The callback URI. */
+  callbackUri?: string;
+  /** SubscriptionId for media streaming */
+  mediaSubscriptionId?: string;
+}
+
+/** Contract model of an ACS call participant */
+export interface CallParticipant {
+  /** Communication identifier of the participant */
+  identifier?: CommunicationIdentifier;
+  /** Is participant muted */
+  isMuted?: boolean;
+}
+
+export interface ServerCallLocator {
```

Comment:
descriptions for locators? I think these are somewhat complex concept that may require some explanations